### ISSUE TITLE
您好，想知道run_sequnce_labeling代码中训练过程一开始的loss大概是多少？

### ISSUE THREAD

**Issue by troublemaker-r** [yuanxiaosc/Entity-Relation-Extraction#40]
正在用pytorch复现，在这里遇了疑问，评价指标达不到您的效果，比较差。
`loss = 0.5 * predicate_loss + token_label_loss`
因为tensorflow看不到中间过程是具体输出，想请教您一下。

**Comment by yuanxiaosc** [comment]
可以用TensorFlow对应的tensorboard查看训练过程，目前TensorFlow 2 已经推出，建议你直接使用 https://github.com/huggingface/transformers 来构建
